occurrences/factories.py

```diff
@@ -11,6 +12,14 @@
 from organisations.factories import OrganisationFactory, PersonFactory
 
 
+class LanguageFactory(factory.django.DjangoModelFactory):
+    id = factory.Faker("pystr", max_chars=10)
+    name = factory.Faker("text", max_nb_chars=20)
+
+    class Meta:
+        model = Language
+
```

Comment:
Should we add some default sorting here so FI SV and EN will be always on top?

Reply:
I had made a ticket about that, but when I started to implement it, I started thinking that API should not make such a decision for UI. I made that feature to provider's UI instead and it is included in https://github.com/City-of-Helsinki/palvelutarjotin-admin/pull/156.